OKR.md

```diff
@@ -0,0 +1,50 @@
+# Quarterly Objectives and Key Results
+
+We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
+
+## 2018 Q2
+
+> Work in Progress
+
+### The daemon is stable and so core is reliable
+
+- `PX` - OWNER - The uncaught errors and memory usage are under control and a daemon can run for a week non stop
+- `PX` - OWNER - One or more js-ipfs daemons are part of the IPFS infrastructure
+- `PX` - OWNER - js.ipfs.io exposes an IPFS Gateway provided by a js-ipfs daemon
+
+### The core can handle vast amounts of data
+
+- `PX` - OWNER - The MFS is implemented in js-ipfs
+- `PX` - OWNER - npm on IPFS is used as a test vector of js-ipfs capabilities
+- `PX` - OWNER - js-ipfs in the browser can load a dataset of at least 100Gb (create archives.js.ipfs.io)
+- `PX` - OWNER - js-ipfs in Node.js can load a dataset of at least 1TB (use npm to prove it)
+- `PX` - OWNER - A S3 IPFS Repo backend exists and it is easily pluggable on `new IPFS`
+- `PX` - OWNER - The PIN API is finished and merged.
+
+### The APIs are more ergonomic and developer productivity is off the charts
+
+- `PX` - OWNER - Complete the first milestone of the Awesome Endeavour - Flow
+- `PX` - OWNER - Roadmap and Kickstart the second milestone for the Awesome Endeavour - Flow
+- `PX` - OWNER - Make js-ipfs uglify friendly again
+- `PX` - OWNER - Every env test run is parallelized in CI
```

Comment:
@VictorBjelkholm This one could also be tracked by the QA Working Group OKR list, I believe you are already onto it, please confirm.

Reply:
Not sure what this means either. That the different envs (windows, macos, linux) are run in parallel? In that case, this is already done

Reply:
@VictorBjelkholm not the browsers. Running Browser tests still only happens in sequence and only one browser is tested.